src/olympia/devhub/views.py

```diff
@@ -661,7 +662,8 @@ def file_validation(request, addon_id, addon, file_id):
 
     validate_url = reverse('devhub.json_file_validation',
                            args=[addon.slug, file_.id])
-    file_url = reverse('files.list', args=[file_.id, 'file', ''])
+    file_url = code_manager_url(
+        'browse', addon_id=addon.pk, version_id=file_.version.pk)
```

Comment:
it's a shame you can't just pass in a `Version` instance and get the `.pk` and `.addon.pk` from there but I suppose it's used in other places where you only have ids.

Reply:
That's an interesting idea. It might be possible, but we already have other code calling and using the `code_manager_url` function, and I wouldn't want to add changes to that to this patch. It's possible you're right, and we might be able to do this, but I'm not sure it's worth a follow-up. WDYT?

Reply:
I looked and it was used in a template used for scanner results that seemed to be literal version ids so you'd need to go back into the view code and check that the version objects are available (having to do a db lookup just for this is a waste).  So, yeah, probably not worth logging for a follow-up.